jina/importer.py

```diff
@@ -26,62 +92,32 @@ def import_classes(namespace: str, targets=None,
 
     import os, re
```

Comment:
Unrelated to this PR, but why is this done here? If there is a magical reason, please add a comment about it in the code.

Reply:
removed.